client/src/components/AddMovieForm/AddMovieForm.spec.js

```diff
@@ -0,0 +1,31 @@
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import AddMovieForm from './AddMovieForm'
+
+describe('AddMovieForm', () => {
+  it('renders', () => {
+    render(<AddMovieForm />)
+
+    const h2element = screen.getByRole('heading', { level: 2 })
+    expect(h2element).toBeInTheDocument()
+
+    const labelItem = screen.getByLabelText('TMDB Id')
+    expect(labelItem).toBeInTheDocument()
+
+    const buttonElement = screen.getByRole('button')
+    expect(buttonElement).toBeInTheDocument()
+  })
+
+  // Does not work yet:
+  // Cannot find module '@testing-library/dom'
+  // from 'node_modules/@testing-library/user-event/dist/click.js'
+  it('has an onSubmit', () => {
+    render(<AddMovieForm />)
+
+    const mockOnSubmit = jest.fn()
+
+    const buttonElement = screen.getByRole('button')
+    userEvent.click(buttonElement)
+    expect(mockOnSubmit).toHaveBeenCalled(1)
+  })
```

Comment:
Since the submit event is happening in your component and you don't have a callback function, this test won't test anything. You could think about adding a callback function and doing the `GET` and `POST` request in your app.